widgetastic_manageiq.py

```diff
@@ -878,3 +878,155 @@ def last_page(self):
     @property
     def items_amount(self):
         return self.paginator.page_info()[1]
+
+
+class Stepper(Widget):
+    """ A CFME Stepper Control
+
+    .. code-block:: python
+
+        stepper = Stepper(locator='//div[contains(@class, "timeline-stepper")]')
+        stepper.increase()
+    """
+    ROOT = ParametrizedLocator('{@locator}')
+    INPUT_CTL = './/input[contains(@class, "bootstrap-touchspin")]'
+
+    def __init__(self, parent, locator, logger=None):
+        Widget.__init__(self, parent=parent, logger=logger)
+
+        self.locator = locator
+
+    @property
+    def minus_button(self):
+        return Button(self, '-')
```

Comment:
@izapolsk use the View base class instead of Widget and it will work. Then the first line in `__init__ ` should refer to View instead of Widget. And that should make it work.